ml-agents/mlagents/trainers/tests/torch/test_simple_rl.py

```diff
@@ -39,13 +39,15 @@
 SAC_TORCH_CONFIG = sac_dummy_config()
 
 
+@pytest.mark.check_environment_trains
```

Comment:
Mark and disable the simple_rl tests since they broken easily and not the main purpose of the GPU tests.
The rest of the tests should be able to capture the non-contiguous errors and device errors.